app/controllers/admin/enterprise_groups_controller.rb

```diff
@@ -15,8 +18,22 @@ def move_down
       redirect_to main_app.admin_enterprise_groups_path
     end
 
+    protected
+
+    def build_resource_with_address
+      enterprise_group = build_resource_without_address
+      enterprise_group.address = Spree::Address.new
+      enterprise_group.address.country = Spree::Country.find_by_name(ENV['DEFAULT_COUNTRY'])
```

Comment:
`Spree::Config.default_country_id` is the One Source of Truth for the default country.